### ISSUE TITLE
add a testable "convergence" criteria

### ISSUE THREAD

**Issue by stites** [Sentenai/reinforce#11]
This would allow us to start doing some convergence testing for #10

**Comment by msaroufim** [comment]
Could you elaborate on this? You mean checking if the diff in training error between two successive runs is less than some epsilon? Or do you have something else in mind?

**Comment by stites** [comment]
Yeah, that was the rough idea -- maybe looking at the last N-run (or doing some sort of significance testing RE: [Deep Reinforcement Learning that Matters](https://arxiv.org/abs/1709.06560)).
